terminal.py

```diff
@@ -196,26 +196,51 @@ def main():
             b_quit = mill.papermill_menu()
 
         elif ns_known_args.opt == "ba":
-            b_quit = ba_controller.menu(s_ticker, s_start)
+            b_quit = ba_controller.menu(
+                s_ticker.split(".")[0] if "." in s_ticker else s_ticker, s_start
+            )
 
         elif ns_known_args.opt == "res":
-            b_quit = rm.res_menu(s_ticker, s_start, s_interval)
+            b_quit = rm.res_menu(
+                s_ticker.split(".")[0] if "." in s_ticker else s_ticker,
+                s_start,
+                s_interval,
+            )
 
         elif ns_known_args.opt == "ca":
             b_quit = ca_controller.menu(df_stock, s_ticker, s_start, s_interval)
 
         elif ns_known_args.opt == "fa":
-            b_quit = fam.fa_menu(s_ticker, s_start, s_interval)
+            b_quit = fam.fa_menu(
+                s_ticker.split(".")[0] if "." in s_ticker else s_ticker,
```

Comment:
So this could cause an issue if stock.XYX has stock in ( for example ) both NYSE and .NS ?

Reply:
This is not really because a ticker is an exchange other than NYSE, NASDAQ. But because the way you load them using yahoo finance is by adding that .NS. 

Then when you are in one of this menus, if you search for google mentions of a ticker, instead of searching for SBIN you were searching for SBIN.NS. So, there was nothing really being found.